examples/get-started/first_figure.py

```diff
@@ -0,0 +1,156 @@
+# script instead of a notebook, use ``fig.show(method="external)`` to display
+# the figure.
+
+fig.show()
+
+###############################################################################
+# Color the land and water
+# ------------------------
+#
+# This figure plots all of the coastlines in the given region, but it does not
+# indicate where the land and water are. Color values can be passed to ``land``
+# and ``water`` to set the colors on the figure.
+#
+# When plotting colors in PyGMT, there are multiple
+# :gmt-docs:`color codes <gmtcolors.html>`, that can be used. This includes
+# standard GMT color names (like ``skyblue``), R/G/B levels (like ``0/0/255``),
+# a hex value (like ``#333333``), and a graylevel (like ``50``). For this
+# example, GMT color names are used.
+
+fig = pygmt.Figure()
+fig.coast(
+    region=[-69, -68, 43.75, 44.75],
+    shorelines=True,
+    land="lightgreen",
+    water="lightblue",
+)
+fig.show()
+
+###############################################################################
+# Set the projection
+# ------------------
+#
+# This figure now has its colors set, but there is not projection or size
+# set for the map. Both of these values are set using the ``projection``
+# parameter.
+#
+# The appropriate projection varies for the type of map. The available
+# projections are explained in the "Projections" gallery. For this example,
+# the Mercator projection is set using "M". The width of the figure will be
+# 10 centimeters, as set by "10c". The map size can also be set in inches
+# using "i" (e.g. a 5 inch wide Mercator projection would use "M5i").
+
+fig = pygmt.Figure()
+fig.coast(
+    region=[-69, -68, 43.75, 44.75],
+    shorelines=True,
+    land="lightgreen",
+    water="lightblue",
+    projection="M10c",
+)
+fig.show()
+
+###############################################################################
+# Add a frame
+# -----------
+#
+# Now that the map's colors, projection, and size have been set, the region
+# that is being displayed is not apparent. A frame can be added to the
+# latitude and longitude of the region.
+#
+# The ``frame`` parameter is used to add a frame to the figure. For now, it
+# will be set to ``True`` to use default settings, but later tutorials will
+# show how ``frame`` can be used to customize the axes, gridlines, and titles.
+
+fig = pygmt.Figure()
+fig.coast(
+    region=[-69, -68, 43.75, 44.75],
+    shorelines=True,
+    land="lightgreen",
+    water="lightblue",
+    projection="M10c",
+    frame=True,
+)
+fig.show()
+
+###############################################################################
+# Additional exercises
+# --------------------
+#
+# This is the end of the first tutorial. Here are some additional exercises
+# for the concepts that were discussed:
+#
+# 1. Make a map of the Canary Islands. Use a tool like Google Maps to
+# determine the coordinates for the ``region`` argument.
```

Comment:
I would prefer a different additional exercise for this first example because the next DCW release will include ISO codes for regions such as the Canary Islands meaning that users should no longer need to use trial and error to get the region correct. Are there lakes or rivers in this area, where something like 'Plot lakes using ``lakes`` parameter, passing a color such as ``darkblue`` as the argument' could be used?

Reply:
Slight variation of this. I made the first exercise one for plotting using the ISO code, and then added an exercise to change the color of the lakes.